stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml

```diff
@@ -33,7 +33,7 @@ params:
       items: {type: string}
       description: Taxon edge fields to keep in the results
       default: null
-query_prefix: WITH ws_object_version
+query_prefix: WITH ws_object_version, ws_type_version
 query: |
   let count = COUNT(
```

Comment:
this seems to be returning a count of all object versions attached to the taxon, including objects to which the user has no access. That seems wrong, naively.

Reply:
This is what the frontend/product folks requested. A little bit of a security leak, but I think negligible :man_shrugging: